pkgs/applications/blockchains/bitcoin.nix

```diff
@@ -1,57 +1,73 @@
-{ stdenv, fetchurl, pkgconfig, autoreconfHook, openssl, db48, boost, zeromq, rapidcheck, hexdump
-, zlib, miniupnpc, qtbase ? null, qttools ? null, wrapQtAppsHook ? null, utillinux, python3, qrencode, libevent
-, withGui }:
+{ stdenv
+, fetchurl
+, pkgconfig
+, autoreconfHook
+, db48
+, boost
+, zeromq
+, rapidcheck
```

Comment:
We have removed our [rapidcheck integration and tests](https://github.com/bitcoin/bitcoin/pull/18514), so you should be able to drop this dependency.

Reply:
Thanks!